hooks/pre_migrations/11-migrate_legacy_katello_config.rb

```diff
@@ -13,7 +13,7 @@
     scenario = Kafo::Configuration.new(scenario_file)
 
     # copy over config and answers
-    scenario.migrate_configuration(legacy_config)
+    scenario.migrate_configuration(legacy_config, :skip => [:log_dir, :log_name])
```

Comment:
I'm not sure if users tend to touch the logging setup but it is the case we could do conditional migration if the value differs from our old defaults:
```ruby
scenario.app[:log_dir] = legacy_config.app[:log_dir] unless legacy_config.app[:log_dir] == '/var/log/katello-installer'
scenario.app[:log_name] = legacy_config.app[:log_name] unless legacy_config.app[:log_name] == 'katello-installer.log'
```